### PR TITLE
Fix `syscall` warning in Bazel Build

### PR DESCRIPTION
**Pull request by ArthurBandaryk**
This commit resolves [#717](https://github.com/google/glog/issues/717) issue.

### CODE REVIEW

File: bazel/glog.bzl [hunk 1]
```diff
@@ -92,6 +92,8 @@ def glog_library(namespace = "google", with_gflags = 1, **kwargs):
     darwin_only_copts = [
         # For stacktrace.
         "-DHAVE_DLADDR",
+        "-DHAVE_PTHREAD_THREADID_NP",
```
Comment (drigz) [inline bazel/glog.bzl:95]: Could you add a comment `# Avoid deprecated syscall()`? Otherwise someone might think this is needed for a stacktrace.

**Review by drigz — CHANGES_REQUESTED** [review]
Thanks for the PR!

File: bazel/glog.bzl [hunk 2]
```diff
@@ -92,6 +92,9 @@ def glog_library(namespace = "google", with_gflags = 1, **kwargs):
     darwin_only_copts = [
         # For stacktrace.
         "-DHAVE_DLADDR",
+        # Avoid deprecated syscall().
+        "-DHAVE_PTHREAD_THREADID_NP",
+        "-Werror",
```
Comment (sergiud) [inline bazel/glog.bzl:97]: I'm not sure it's a good idea to add the flag here and for macOS only. @drigz Is there a way to enable `-Werror` exclusively for CI builds?

**Comment by drigz** [comment]
@ArthurBandaryk I'm not seeing a CLA signed for your GitHub username, is it possible you signed with a different username?

**Review by drigz — APPROVED** [review]

**Pull request merged**